apprise/plugins/one_signal.py

```diff
@@ -82,7 +83,7 @@ class NotifyOneSignal(NotifyBase):
     setup_url = 'https://github.com/caronc/apprise/wiki/Notify_onesignal'
 
     # Notification
-    notify_url = "https://onesignal.com/api/v1/notifications"
+    notify_url = "https://api.onesignal.com/notifications"
```

Comment:
I don't see this in [the documentation](https://documentation.onesignal.com/reference/create-notification).  All of the provided `curl` requests use `https://onesignal.com/api/v1/notifications`

Reply:
![image](https://github.com/user-attachments/assets/7aa4061f-1bba-4b0c-9a32-13853b5b36e2)
This is taken straight from their official website. I've also asked their support (as I'm on a premium plan) and they confirmed that `v1` is going to be deprecated soon.